src/main/java/com/conveyal/datatools/common/utils/SparkUtils.java

```diff
@@ -92,31 +92,31 @@ public static String formatJSON(String message, int code, Exception e) {
     /**
      * Wrapper around Spark halt method that formats message as JSON using {@link SparkUtils#formatJSON}.
      */
-    public static void haltWithMessage(Request request, int statusCode, String message) throws HaltException {
-        haltWithMessage(request, statusCode, message, null);
+    public static void logMessageAndHalt(Request request, int statusCode, String message) throws HaltException {
+        logMessageAndHalt(request, statusCode, message, null);
     }
 
     /**
      * Wrapper around Spark halt method that formats message as JSON using {@link SparkUtils#formatJSON}.
      * Extra logic occurs for when the status code is >= 500.  A Bugsnag report is created if
      * Bugsnag is configured.
      */
-    public static void haltWithMessage(
+    public static void logMessageAndHalt(
         Request request,
         int statusCode,
         String message,
         Exception e
     ) throws HaltException {
+        // Note that halting occurred, also print error stacktrace if applicable
+        if (e != null) e.printStackTrace();
+        LOG.info("Halting with status code {}", statusCode);
```

Comment:
@evansiroky, thanks for this change, but I was originally thinking that we would also log the `message` arg unconditionally as well (i.e., combining the log statement at line 115 with this log statement). I feel like logging just the statusCode leaves out some info that could be helpful in debugging.